### ISSUE TITLE
Validation set?

### ISSUE THREAD

**Issue by kmyi** [daitao/SAN#36]
Hello,

I've been searching the codebase for a validation routine, and it seems the current framework is not using a separate validation set. Are you using the test set as the validation set?

Thanks,
Kwang

**Comment by kmyi** [comment]
Sorry. Nevermind. Found that you have a validation set reserved in the div2k. ;)

**Comment by shazib-summar** [comment]
Could you please tell me your PyTorch and Python version?
